pkg/server/debug/server.go

```diff
@@ -221,28 +220,33 @@ func (ds *Server) RegisterWorkloadCollector(stores *kvserver.Stores) error {
 	return nil
 }
 
+// GetEngineMetrics generates a string combining debug metrics for all of the provided storage engines.
+func GetEngineMetrics(engines []storage.Engine) (string, error) {
```

Comment:
nit: maybe we should call this `GetLSMStats` instead

Reply:
Agreed, this is [fixed](https://github.com/cockroachdb/cockroach/pull/125865/files#diff-1b7def1db9a92b3ed25a4dad6a300b12d82a318cd4df2a5d4fdde334403fff69R225)